tajo-catalog/tajo-catalog-server/src/main/java/org/apache/tajo/catalog/store/XMLCatalogSchemaManager.java

```diff
@@ -675,40 +654,38 @@ protected void validateSQLObject(List<SQLObject> queries, SQLObject testQuery) {
     }
     
     protected void mergeExistQueries(List<SQLObject> queries) {
-      for (SQLObject query: queries) {
+      queries.forEach(query -> {
         validateSQLObject(queries, query);
-        
+
         targetStore.addExistQuery(query);
-      }
+      });
     }
     
     protected void mergeDropStatements(List<SQLObject> queries) {
-      for (SQLObject query: queries) {
+      queries.forEach(query -> {
         validateSQLObject(queries, query);
-        
+
         targetStore.addDropStatement(query);
-      }
+      });
     }
     
```

Comment:
I think these two methods can be removed and merged as lambda itself unless it is called from many places.